src/ui/components/search/searchcomponent.js

```diff
@@ -138,6 +138,15 @@ export default class SearchComponent extends Component {
       if (this.queryEl) {
         this.queryEl.value = q;
       }
+      if (q === null) {
+        this.core.globalStorage.set(StorageKeys.VERTICAL_RESULTS, {
```

Comment:
This is not an exhaustive list. There are probably a million filters I'm missing. Do we have a better way of clearing everything for this case? Note: we use `searchState: 'search-complete'` o/w the VerticalResults don't update. (Can use the const later, just hardcoded string for PoC)

Reply:
Thoughts on creating a method setEmptySearch inside global storage that performs the clearing?

Reply:
👍 
put in core because core has most of the logic about which globalStorage keys are changed on search